### PR TITLE
Fix issue 20

### PR DESCRIPTION
**Pull request by nicolasfranck**
First version for #20 

HTTP paths should follow table here: https://github.com/ugent-library/people-service/issues/20#issuecomment-1697390190

Feel free to comment

### CODE REVIEW

File: api/v2/service.go [hunk 1]
```diff
@@ -17,14 +18,156 @@ func NewService(repository models.Repository) *Service {
...
+	}
+	for _, person := range people {
+		res.Data = append(res.Data, *mapToExternalPerson(person))
+	}
+
+	return res, nil
+}
+
+func (s *Service) SuggestPeople(ctx context.Context, req *SuggestPeopleRequest) (*PersonListResponse, error) {
+	people, err := s.repository.SuggestPerson(ctx, req.Query)
```
Comment (netsensei) [inline api/v2/service.go]: Repo method `SuggestPerson` should be renamed to `SuggestPeople` as well.

File: api/v2/service.go [hunk 2]
```diff
@@ -17,14 +18,156 @@ func NewService(repository models.Repository) *Service {
...
+	}
+	for _, org := range organizations {
+		res.Data = append(res.Data, *mapToExternalOrganization(org))
+	}
+
+	return res, nil
+}
+
+func (s *Service) SuggestOrganizations(ctx context.Context, req *SuggestOrganizationsRequest) (*OrganizationListResponse, error) {
+	orgs, err := s.repository.SuggestOrganization(ctx, req.Query)
```
Comment (netsensei) [inline api/v2/service.go]: Repo method `SuggestionOrganization` should become plural. `SuggestOrganizations`.

**Review by netsensei — CHANGES_REQUESTED** [review]

File: api/v2/service.go (hunk 1 above)
Comment (nicolasfranck) [reply api/v2/service.go]: agreed. I was hesitant to change anything else than requested

File: api/v2/service.go (hunk 2 above)
Comment (nicolasfranck) [reply api/v2/service.go]: agreed. I was hesitant to change anything else than requested.

File: api/v2/service.go (hunk 1 above)
Comment (nicolasfranck) [reply api/v2/service.go]: done

File: api/v2/service.go (hunk 2 above)
Comment (nicolasfranck) [reply api/v2/service.go]: done

**Review by netsensei — APPROVED** [review]

**Comment by netsensei** [comment]
@nicolasfranck Okay to merge this. If we need to change anything, let's do it in a follow up issue.

**Pull request merged**